src/main/java/ui/components/pickers/AssigneePickerDialogResponse.java

```diff
@@ -0,0 +1,23 @@
+package ui.components.pickers;
+
+import javafx.scene.control.ButtonType;
+
+import java.util.Optional;
+
+public class AssigneePickerDialogResponse {
```

Comment:
Could this be an inner class? Since it doesn't really exist outside the context of an assignee dialog.